app/config/locale/translations/kn.json

```
@@ -3,30 +3,30 @@
     "settings.locale": "ka",
     "settings.direction": "ltr",
     "emails.sender": "%s ತಂಡ",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "ಖಾತೆ ಪರಿಶೀಲನೆ",
+    "emails.verification.hello": "ನಮಸ್ಕಾರ {{name}}",
+    "emails.verification.body": "ನಿಮ್ಮ ಇಮೇಲ್ ವಿಳಾಸ ಪರಿಶೀಲನೆಗೆ ಈ ಲಿಂಕನ್ನು ಅನುಸರಿಸಿ",
+    "emails.verification.footer": "ನೀವು ಇಮೇಲ್ ವಿಳಾಸ ಪರಿಶೀಲನೆಗೆ ಕೇಳದಿದ್ದರೆ, ಈ ಸಂದೇಶವನ್ನು ನಿರ್ಲಕ್ಷಿಸಿ",
+    "emails.verification.thanks": "ಧನ್ಯವಾದಗಳು",
+    "emails.verification.signature": "{{project}} ತಂಡ",
+    "emails.magicSession.subject": "ಲಾಗಿನ್",
+    "emails.magicSession.hello": "ನಮಸ್ಕಾರ,",
+    "emails.magicSession.body": "ಲಾಗಿನ್ ಮಾಡಲಿಕ್ಕೆ ಈ ಲಿಂಕನ್ನು ಅನುಸರಿಸಿ",
+    "emails.magicSession.footer": "ನೀವು ಈ ಇಮೇಲನಿಂದ ಲಾಗಿನ್ ಮಾಡಲು ಕೇಳದಿದ್ದರೆ, ಈ ಸಂದೇಶವನ್ನು ನಿರ್ಲಕ್ಷಿಸಿ",
+    "emails.magicSession.thanks": "ಧನ್ಯವಾದಗಳು",
+    "emails.magicSession.signature": "{{project}} ತಂಡ",
+    "emails.recovery.subject": "ಗುಪ್ತಪದ ಮರುಹೊಂದಿಸಿ",
+    "emails.recovery.hello": "ನಮಸ್ಕಾರ {{name}}",
+    "emails.recovery.body": "ನಿಮ್ಮ {{project}} ಗುಪ್ತಪದವನ್ನು ಮರುಹೊಂದಿಸಲು ಈ ಲಿಂಕನ್ನು ಅನುಸರಿಸಿ",
+    "emails.recovery.footer": "ನೀವು ಗುಪ್ತಪದವನ್ನು ಮರುಹೊಂದಿಸಲು ಕೇಳದಿದ್ದರೆ, ಈ ಸಂದೇಶವನ್ನು ನಿರ್ಲಕ್ಷಿಸಿ",
+    "emails.recovery.thanks": "ಧನ್ಯವಾದಗಳು",
+    "emails.recovery.signature": "{{project}} ತಂಡ",
+    "emails.invitation.subject": "%s ತಂಡಕ್ಕೆ %sರಲ್ಲಿ ಆಹ್ವಾನ",
```

```suggestion
    "emails.invitation.subject": "%s ತಂಡಕ್ಕೆ %s ರಲ್ಲಿ ಆಹ್ವಾನ",
```

Please ensure the spaces, and punctuation are exactly the same as `en.json`